app/helpers/foreman_ansible/ansible_reports_helper.rb

```diff
@@ -106,5 +115,19 @@ def parsed_message_json(log)
       Foreman::Logging.exception('Error while parsing ansible message json', e)
       {}
     end
+
+    def add_diff(message, title, result)
+      diff = if result['report_diff'].blank?
+               _('No Diff')
+             else
+               # Add '\n' before the unified diff as the Javascript displaying the
+               # diff expects this.
+               link_to(_('Show Diff'), '#', data: { diff: "\n#{result['report_diff']}", title: title }, onclick: 'tfm.configReportsModalDiff.showDiff(this);')
+             end
+      # diff is already HTML safe and should not be escaped again. Hence escape
+      # the contents of message and concatenate it with diff and
+      # declare it HTML safe.
+      message.present? ? (h(message) + ' ' + diff).html_safe : diff # rubocop:disable Rails/OutputSafety
```

Comment:
Looks like RuboCop is unhappy with my suggestion and wants:
```suggestion
      message.present? ? "#{h(message)} #{diff}".html_safe : diff # rubocop:disable Rails/OutputSafety
```